subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/resolver/KotlinScriptDependenciesResolverTest.kt

```diff
@@ -284,7 +278,7 @@ class KotlinScriptDependenciesResolverTest : AbstractKotlinIntegrationTest() {
     @Test
     fun `do not report file warning on script compilation failure in currently edited script`() {
         // because the IDE already provides user feedback for those
-        assumeNonEmbeddedGradleExecuter()
+//        assumeNonEmbeddedGradleExecuter()
```

Comment:
:question: if that test works in embedded mode, let's get rid of that commented code